IbisPlugins/GPU_RigidRegistration/CMakeLists.txt

```diff
@@ -24,10 +22,29 @@ IF( NOT IBIS_USE_ELASTIX )
   message( SEND_ERROR "GPU_RigidRegistration plugin require Elastix library. Please enable IBIS_USE_ELASTIX.\n" )
 ENDIF()
 
-add_subdirectory( itkRegistrationOpenCL )
+#--------------------------------------------------
+# Fetch and setup build for GPURigidRegistration lib
+#--------------------------------------------------
+include(FetchContent)
+#FetchContent_Declare(
+#  GPURigidRegistrationLib
+#  GIT_REPOSITORY https://github.com/drouin-simon/GPURigidRegistrationLib.git
+#  GIT_TAG fix-build-mac-remove-vtk-mni-from-lib )
+
+FetchContent_Declare(
+  GPURigidRegistrationLib
+  GIT_REPOSITORY https://github.com/IbisNeuronav/GPURigidRegistrationLib.git
+  GIT_TAG main )
```

Comment:
By default the binary directory is ${CMAKE_BINARY_DIR}/_deps, I added the following lines to make it more consistent with other dependencies:

```
set(FETCHCONTENT_BASE_DIR ${IBIS_EXTERNAL_DEPENDENCIES_DIR}/GPUReg CACHE PATH "Directory under which to collect all populated content" FORCE)

FetchContent_Declare(
  GPURigidRegistrationLib
  GIT_REPOSITORY https://github.com/IbisNeuronav/GPURigidRegistrationLib.git
  GIT_TAG main 
  SOURCE_DIR ${FETCHCONTENT_BASE_DIR}/src
  BINARY_DIR ${FETCHCONTENT_BASE_DIR}/build
  SUBBUILD_DIR ${FETCHCONTENT_BASE_DIR}/subbuild
  )
```